### PR TITLE
Fix double counting while populating `promscale_sql_database_chunks_count` metric

### PR DESCRIPTION
**Pull request by arajkumar**
### Problem statment
`PromscaleCompressionLow` alert fires regardless of a good compression ratio.

### Root cause

SQL query which populates `promscale_sql_database_chunks_count` is a sum of the below metrics,
1) Uncompressed Chunks
2) Proxy Chunks which points to Compressed Chunks
3) Compressed Chunks

However the total chunk count should be just (1) + (3), because (2) is already pointing to (3) and which leads to double counting.

### Solution

Fix the SQL query to consider entries which has `compressed_chunk_id` as null and this will be true for both (1) and (3).

Signed-off-by: Arunprasad Rajkumar <ar.arunprasad@gmail.com>

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [x] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

File: pkg/tests/end_to_end_tests/database_metrics_test.go [hunk 1]
```diff
@@ -58,6 +66,59 @@ func TestDatabaseMetrics(t *testing.T) {
 
 		chunksCreated = getMetricValue(t, "chunks_created")
 		require.Equal(t, chunksCreated, float64(3))
+		chunksCount = getMetricValue(t, "chunks_count")
+		require.Equal(t, chunksCount, float64(3))
```
Comment (alejandrodnm) [inline pkg/tests/end_to_end_tests/database_metrics_test.go:70]: nit:
```suggestion
		require.Equal(t, float64(3), chunksCount)
```

**Review by alejandrodnm — APPROVED** [review]

File: pkg/tests/end_to_end_tests/database_metrics_test.go (hunk 1 above)
Comment (arajkumar) [reply pkg/tests/end_to_end_tests/database_metrics_test.go]: What a blooper :P

File: docs/runbooks/PromscaleCompressionLow.md [hunk 2]
```diff
@@ -12,9 +12,10 @@ High disk usage by Promscale database
 1. Open Grafana and navigate to Promscale dashboard
 2. Go to Database section and see `Compressesd chunks ratio`. If you see a ratio of < 10% then compression is not adequate in your system
 3. Open psql
-4. Check number of uncompressed chunks: `select count(*)::bigint from _timescaledb_catalog.chunk where dropped=false and compressed_chunk_id=null;`
-5. Check number of maintenancec jobs: `select count(*) from timescaledb_information.jobs where proc_name = 'execute_maintenance_job'`
-6. Run the following debugging query:
+4. Check total number of chunks: `select count(*)::bigint from _timescaledb_catalog.chunk where dropped=false and compressed_chunk_id is null;`
+5. Check total number of compressed chunks: `select count(*)::bigint from _timescaledb_catalog.chunk where dropped=false and compressed_chunk_id is not null;`
+6. Check number of maintenancec jobs: `select count(*) from timescaledb_information.jobs where proc_name = 'execute_maintenance_job'`
```
Comment (JamesGuthrie) [inline docs/runbooks/PromscaleCompressionLow.md:17]:
```suggestion
6. Check number of maintenance jobs: `select count(*) from timescaledb_information.jobs where proc_name = 'execute_maintenance_job'`
```

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]
minor nit

**Review by JamesGuthrie — APPROVED** [review]

**Review by antekresic — APPROVED** [review]

File: pkg/tests/end_to_end_tests/database_metrics_test.go (hunk 1 above)
Comment (alejandrodnm) [reply pkg/tests/end_to_end_tests/database_metrics_test.go]: It has happened to me soooo many times.

**Pull request merged**